packages/service-library/src/servicelib/bytes_iters/_stream_zip.py

```diff
@@ -42,6 +45,7 @@ async def get_zip_bytes_iter(
         sum(bytes_streamer.data_size for _, bytes_streamer in archive_entries)
     )
     description = f"files: count={len(archive_entries)}, size={total_stream_lenth.human_readable()}"
+    _logger.info("Archiving %s", description)
```

Comment:
This is actually returning a generator not an iterator right?
the name of the function is confusing, the logging, would also be probably better served by using `with log_context` instead.

Reply:
you already have loggers around this. why add yet another one?